tox.ini

```
@@ -92,6 +92,38 @@ skip_install = true
 allowlist_externals =
   git
 
+
+[testenv:preview]
```

Other envs also have this suffix:
```suggestion
[testenv:preview-docs]
```
Let's have it here too, for consistency.